documentation/book/proc-deploying-cluster-operator.adoc

```diff
@@ -2,11 +2,12 @@
 //
 // assembly-cluster-operator.adoc
 
-[id='deploying-cluster-operator-kubernetes-{context}']
-= Deploying the Cluster Operator to {KubernetesName}
+[id='deploying-cluster-operator-{context}']
+= Deploying the Cluster Operator to Kubernetes
 
 .Prerequisites
 
+* On OpenShift a user with `cluster-admin` role needs to be used, for example, `system:admin`.
```

Comment:
That's not clear because `system:admin` is not a thing on kube (is it?)

Reply:
The `system:` users and groups are in pure Kubernetes as well. I'm not sure whether exactly `system:admin` exists - it might on some distros. I think the reference to `system:admin` is dubious on its own. Apart from the Minishift and `oc cluster up` you cannot login as `system:admin` just like that.

Reply:
Really the main problem here is for me that this gives the impression you do not need these rights on Kube which is not true.

Reply:
Yes, I agree we need a consistent way to convey when extra rights are needed for a procedure. But I'm not sure we should do that by talking about `system:admin` or even admin-like roles, because the whole point of RBAC is that this stuff is very flexible/configurable. So I would prefer a form of words something like:

NOTE: This procedure requires use of a user account which is able to create `CustomResourceDefinitions`, `ClusterRoles` and`ClusterRoleBindings`. For more information see xref:foo[].

And then in `foo` we explain about system:admin, and how to add those privs to a normal user account etc. I think that makes it much easier to dip into procedures like this and emerge successful, even if you have to follow a few links to do so. It also makes it clearer exactly which rights are needed, which makes it easier for users when they go asking their cluster admin to do this for them.

Reply:
I did this in [this commit](https://github.com/strimzi/strimzi-kafka-operator/pull/1701/commits/55a33f8905130f93a3c4bc680b68d12cfdc78082) if you want to take a look.

Reply:
Sorry, I had to rebase for conflict, try [this commit](https://github.com/strimzi/strimzi-kafka-operator/pull/1701/commits/3b977f1031e385681986d5d98d1c2f6c7cfeff93) @scholzj

Reply:
@tombentley Sorry, but I do not think this is the right place to add it. The note in `proc-deploying-cluster-operator.adoc` is good. But the Strimzi administrators chapter is specifically about the users who are unable to install the CO but are allowed to manage the resources - i.e. the Kafka, KafkaUser etc. I think your changes there mix two different things together and might be confusing.

Reply:
You're right. Damn.